token-metadata/program/src/processor/delegate/delegate.rs

```diff
@@ -97,6 +91,17 @@ pub fn delegate<'a>(
         } => Some((TokenDelegateRole::Staking, amount, authorization_data)),
         // Standard
         DelegateArgs::StandardV1 { amount } => Some((TokenDelegateRole::Standard, amount, &None)),
+        // LockedTransfer
```

Comment:
I agree enums must be kept in same order to prevent breaking serialization, but this section is just a match statement so could be in any order.  I changed the order of the match statement to look be more in the order of the enum.

I did not change any enum order, but I did change the name of two metadata delegates, and appended some to the end.  For the two that I changed, Sam ran a script to make sure nobody was using them yet.  The appended ones should be fine right?

Reply:
Ah my bad, IDK why I thought this was the definition.

Reply:
Its all in the same vicinity in the PR.  Overall this is important issue to be looking out for so glad you are looking for that issue in general.  Thanks!